src/Moryx.Runtime.Maintenance.Web/Moryx.Runtime.Maintenance.Web.csproj

```diff
@@ -78,9 +78,6 @@
     </EmbeddedResource>
   </ItemGroup>
   <ItemGroup>
-    <EmbeddedResource Include="..\Moryx.Runtime.Maintenance.Web.UI\dist\bundle.js">
-      <Link>wwwroot\bundle.js</Link>
-    </EmbeddedResource>
```

Comment:
This lines must be existent because the bundle.js must be embedded into the project to expose it within the web service. I saw that the bundle.js placeholder was missing in the repo. This was a mistake while making the repo open source compatible. I added the file again, so you can revert these changes